force-app/main/default/classes/StringUtilTest.cls

```diff
@@ -0,0 +1,14 @@
+@isTest
+private with sharing class StringUtilTest {
+
+    @isTest
+    private static void generateRandomString_expectStringReturned() {
+        Integer STRING_SIZE = 10;
+        String testString1 = StringUtil.generateRandomString(STRING_SIZE);
+        System.assertEquals(STRING_SIZE, testString1.length(), 'Expected generated string to be of correct length.');
+
+        String testString2 = StringUtil.generateRandomString(STRING_SIZE);
+
+        System.assertNotEquals(testString1, testString2, 'Expected a different random string to be returned on subsequent calls.');
```

Comment:
Technically this test could fail in the ultra random chance that the two random strings were randomly resulting in the same output. It can be hard to control random.